feature_engine/feature_selection.py

```diff
@@ -88,3 +89,138 @@ def transform(self, X):
         return X
 
 
+class SelectFeatures(TransformerMixin, BaseEstimator):
+    """
+    SelectFeatures()
+
+    Drops constant and quasi constant variables from a dataframe.
+    Constant variables are those variables which show only one value for all the observations.
+    By default, all constant variables are dropped.
+    Quasi-constant variables are those variables which are almost constant. In other words, those features have the same
+    value for large subset of total observations.
+    This transformer works for both, numerical and categorical variables.
+
+    Parameters
+    ----------
+
+    drop_quasi_constant_features : bool, default=False
+        Whether to drop quasi constant features
```

Comment:
I don't think we need this parameter. From the threshold the user can indicate if it will be constant or quasi_constant as well. If threshold is 1, then drops only constant. If threshold is smaller than 1, then quasi_constant as well. 
The less parameters the easier to understand and use

Reply:
Yes, makes sense. Fixed.